cernopendata/modules/fixtures/data/articles/welcome/welcome.md

```diff
@@ -1,14 +1,12 @@
-
-With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal.
+With the growing demand for open research data and an increased interest in the CERN Open Data portal, we have been working hard on modernising our web presence and services. Here, you can learn more about the changes we have made and how you can use our updated portal.
 
 <p align="center">
 <img src="/static/articles/welcome/header-image.png" width="70%"></p>
 
-CERN Open Data provides content for both education and research. We aim to support high school students and teachers as well as university students and professors. For that purpose, the new design of CERN Open Data is article-driven. Exciting [guides](/search?type=Documentation&subtype=Guide) introduce to physics analyses and make use of [datasets](/search?type=Dataset). Event displays allow to visualise and imagine what happens during collisions of the smallest elements.
+CERN Open Data provides content for both education and research. We aim to support high-school students and teachers as well as university students and professors. For this purpose, the new design of CERN Open Data is article-driven. Detailed [guides](/search?type=Documentation&subtype=Guide) introduce you to physics analyses and make use of [datasets available from this portal](/search?type=Dataset). Event displays allow to visualise collision data and examine what happens when particles interact in high-energy collisions.
+
+Even though enthusiastic members of the wider public as well educators in particular have been on our mind when we redesigned the portal, we did not forget about researchers. It’s quite the opposite actually: our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that data released on CERN Open Data led to the [first publication](https://arxiv.org/abs/1704.05842) of a research paper using LHC data by researchers not belonging to any LHC collaboration. This is truly open science!
 
-Even though the general public and educators have been on our mind when we redesigned the portal, we did not forget about researchers. It’s quite the opposite actually. Our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that data released on CERN Open Data led to the [first publication](https://arxiv.org/abs/1704.05842) of a research paper outside of any LHC collaboration.
+The success of CERN Open Data is made possible through working closely with various research collaborations affiliated with CERN. We provide access to data and resources from the four big LHC collaborations: [ALICE](/about/alice), [ATLAS](/about/atlas), [CMS](/about/cms) and [LHCb](/about/lhcb).
```

Comment:
...affiliated with CERN and external partners.

Reply:
@suenjedt: Do you mean "working closely… with external partners"?

Reply:
How about:

> The success of CERN Open Data is made possible through working closely with various CERN-affiliated research collaborations and with external partners.

(Not sure which of the two interpretations you meant.)

Reply:
Yes, you nailed it!